src/tunnel.cc

```diff
@@ -417,9 +419,6 @@ TunnelStateData::checkRetry()
 void
 TunnelStateData::retryOrBail(const char *context)
 {
-    // Since no TCP payload has been passed to client or server, we may
-    // TCP-connect to other destinations (including alternate IPs).
-
```

Comment:
The above comment should be removed because retryOrBail() preconditions are not the ones listed in the comment. The checkRetry() call below determines when we can retry and when we have to bail. We should not try to re-document that method in its caller.